utils/python/CIME/XML/env_mach_pes.py

```diff
@@ -95,6 +95,8 @@ def get_total_tasks(self, comp_classes):
             pstrid = self.get_value("PSTRID_%s"%comp)
             tt = rootpe + (ntasks - 1) * pstrid + 1
             total_tasks = max(tt, total_tasks)
+        if total_tasks < self.get_value("PES_PER_NODE"): # all PEs fit in 1 node
```

Comment:
If a case has total number of tasks that is fewer than PES_PER_NODE, then it may cause job launch errors: e.g. `aprun -n 8 -N 64`.

Also, rather than adjusting upwards to `-n 64 -N 64`, it's probably better to run with expected total number of MPI tasks with `-n 8 -N 8`.